convert2rhel/backup/files.py

```diff
@@ -182,10 +182,5 @@ def restore(self):
             try:
                 os.remove(self.filepath)
                 loggerinst.info("File {filepath} removed".format(filepath=self.filepath))
-            except OSError as err:
-                # Do not call 'critical' which would halt the program. We are in
-                # a rollback phase now and we want to rollback as much as possible.
-                loggerinst.critical_no_exit("Error(%s): %s" % (err.errno, err.strerror))
-                return
-
-        super(MissingFile, self).restore()
+            finally:
+                super(MissingFile, self).restore()
```

Comment:
`finally` is called even when an exception is raised. How will this affect this overall as this now sets `self.enabled = False`?

Reply:
Yes, you are right, there is no need to have the `finally`. Also, thankfully to this, I found other places where some exception in the `restore` method would be handled and haven't been sent to the `BackupController`